doc/make.py

```diff
@@ -290,15 +284,14 @@ def zip_html(self):
 def main():
     cmds = [method for method in dir(DocBuilder) if not method.startswith("_")]
 
+    joined = ",".join(cmds)
     argparser = argparse.ArgumentParser(
-        description="pandas documentation builder",
-        epilog="Commands: {}".format(",".join(cmds)),
+        description="pandas documentation builder", epilog=f"Commands: {joined}",
     )
+
+    joined = ", ".join(cmds)
```

Comment:
Don't need to redeclare this

Reply:
It's not redeclaration. one of the ```joined``` is with a space and one is without.

```joined = ",".join(cmds)```
```joined = ", ".join(cmds)```